src/ngraph/runtime/cpu/mkldnn_utils.cpp

```diff
@@ -58,7 +58,26 @@ static const mkldnn_version_t* get_mkldnn_version()
 std::map<element::Type, const mkldnn::memory::data_type>&
     runtime::cpu::mkldnn_utils::get_mkldnn_data_type_map()
 {
-    // Mapping from POD types to MKLDNN data types
+// Mapping from POD types to MKLDNN data types
+#if defined(USE_MKLDNN_V1)
+    static std::map<element::Type, const mkldnn::memory::data_type> s_mkldnn_data_type_map = {
```

Comment:
Would `unordered_map` be enough here? It would be more efficient. Same for other places where `map` and `set` are used.